pkg/snyk/client.go

```diff
@@ -60,7 +61,8 @@ func NewClient(ctx context.Context, groupID, token string) (*Client, error) {
 
 func (c *Client) prepareURL(path string) *url.URL {
 	u := *c.baseUrl
-	u.Path = path
+	// Passing in the version separately since it encodes '/' if present in base url
+	u.Path = fmt.Sprintf("%s%s", Version, path)
```

Comment:
We should probably use url.JoinPath() so that duplicate or nonexistent slashes get fixed.
```suggestion
	u.Path = url.JoinPath(Version, path)
```